device/send.go

```diff
@@ -212,81 +213,124 @@ func (device *Device) RoutineReadFromTUN() {
 
 	device.log.Verbosef("Routine: TUN reader - started")
 
-	var elem *QueueOutboundElement
+	var (
+		readErr     error
+		elems       = make([]*QueueOutboundElement, device.BatchSize())
```

Comment:
Factor out repeated calls to BatchSize() like before?

Reply:
Updated in https://github.com/WireGuard/wireguard-go/pull/64/commits/9df282b6057a2c3f73930b4a5a93c11d2d704201.